core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemCreateRequest.java

```diff
@@ -101,7 +109,7 @@ private void checkBranchPath(final RepositoryContext context, final boolean crea
 		
 		// If the branch should be created, it branch should not exist, however 
 		if (create) {
-			final String newBranchPath = Branch.get(parentPath, title);
+			final String newBranchPath = Branch.get(parentPath, getTitle());
```

Comment:
Title is definitely wrong here, it should be ID. Tests should break unless you use the same value for both.